src/components/ScholarshipForm.jsx

```diff
@@ -8,13 +8,42 @@ import {
   StepContent,
   Grid,
   Typography,
+  FormControlLabel,
+  Checkbox,
 } from '@material-ui/core';
 import { makeStyles } from '@material-ui/core/styles';
 import PropTypes from 'prop-types';
 import validationSchema from '../validation/ValidationSchema';
 import ScholarshipAmountField from './ScholarshipAmountField';
 import DatePicker from './DatePicker';
 import FormikTextField from './FormikTextField';
+import FormikMultiSelect from './FormikMultiSelect';
+import FormikAutocomplete from './FormikAutocomplete';
+
+const gradeOptions = [
+  'Middle School',
+  'HS Freshman',
+  'HS Sophomore',
+  'HS Junior',
+  'HS Senior',
+  'College Freshman',
+  'College Sophomore',
+  'College Junior',
+  'College Senior',
+];
+
+const ethnicityOptions = [
+  'American Indian or Alaska Native',
+  'Asian',
+  'Black or African American',
+  'Hispanic or Latino',
+  'Native Hawaiian or Other Pacific Islander',
+  'White',
+];
+
+const schoolOptions = [{ title: 'school1' }, { title: 'school2' }];
+const stateOptions = [{ title: 'CA' }, { title: 'WA' }];
+const majorOptions = [{ title: 'major1' }, { title: 'major2' }];
```

Comment:
Why not just use arrays here?

Reply:
In retrospect this is the sort of pattern you may want for the gradeOptions and ethnicityOptions instead since those could be displayed differently than their values.